action.yml

```diff
@@ -1,16 +1,63 @@
-name: Actions Ecosystem Action Push Tag
-description: Push a Git tag.
-author: Actions Ecosystem
+name: Push Any Git Tag
+description: based on Actions Ecosystem Push Tag - Fixed with additional features.
+author: Actions Ecosystem & theJeff77
```

Comment:
I wouldn't change these. Nor do you need to put your name in as the author just because you PR.